features/step_definitions/all_my_step_definitions_step.rb

```diff
@@ -0,0 +1,25 @@
+Given("the following user exists") do |table|
+    table.hashes.each do |user|
+      User.create(user)
+end
+  end
+  
+  Given("I am on the landing page") do
+    visit '/'
+  end
+  
+  When("I click on the link {string}") do |login|
+    click_on(login)
+  end
+  
+  When("I fill in the {string} with {string}") do |a, b|
+    fill_in(a, :with => b)
+  end
+  
+  Then("I should be on the root page") do
+
+  end
+  
+  Then("I should see message {string}") do |message|
+    expect(page).to have_content message
+  end
```

Comment:
I would place this step definition in a specific file where only assertion steps would be placed. Not a blocker though.